From: Jan Engelhardt <jengelh@inai.de>
Date: Mon, 17 Feb 2014 03:58:37 +0100
Subject: doc: properly use XML entities

(cherry picked from commit 3c8fa0fd0f99996088e42c5d48e164fdae4ada40)

Conflicts:
	man/sd_bus_error.xml
---
 man/localtime.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/localtime.xml b/man/localtime.xml
index b7fd1ba..dbf2379 100644
--- a/man/localtime.xml
+++ b/man/localtime.xml
@@ -55,7 +55,7 @@
         </refnamediv>
 
         <refsynopsisdiv>
-                <para><filename>/etc/localtime</filename> -> <filename>../usr/share/zoneinfo/…</filename></para>
+                <para><filename>/etc/localtime</filename> -&gt; <filename>../usr/share/zoneinfo/…</filename></para>
         </refsynopsisdiv>
 
         <refsect1>
